dev.lasheen.qr.yml

```diff
@@ -0,0 +1,38 @@
+app-id: dev.lasheen.qr
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+sdk: org.freedesktop.Sdk
+command: qr
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --socket=pulseaudio
+  - --share=network
+modules:
+  - name: qr
+    buildsystem: simple
+    only-arches:
+      - x86_64
+    build-commands:
+      - 'mkdir -p qr'
+      - 'tar -xf qr-Linux-Portable.tar.gz -C qr'
```

Comment:
if you set the type `archive` you don't need that.

Reply:
Thanks for the tip. I made some changes. Are they any good?